libgalois/include/galois/graphs/LS_LC_CSR_Graph.h

```diff
@@ -242,6 +249,7 @@ class LS_LC_CSR_Graph : private boost::noncopyable {
     {
       m_edges[0].resize(0);
       swap(m_edges[0], m_edges[1]);
+      m_edges_tail.store(0, std::memory_order_seq_cst);
```

Comment:
why is this `seq_cst` at the very least use `memory_order_acq_rel`
but I don't think you need that here.

Reply:
sure, in fact I think relaxed is ok because of the lock.